composer.lock

```diff
@@ -2085,16 +2085,16 @@
         },
         {
             "name": "guzzlehttp/guzzle",
-            "version": "7.4.2",
+            "version": "7.4.4",
```

Comment:
Seems like you have a conflict or a rebase problem because I'm co-authored on this modification and it was already merged in this PR https://github.com/PrestaShop/PrestaShop/pull/29274